src/_shared/scss/_core.scss

```diff
@@ -112,7 +112,7 @@ a.u-faux-block-link__overlay {
 .button {
     border: 0;
     border-radius: 1000px;
-    font: bold 13px/20px $f-sans-serif-text;
+    font: bold 13px/0px $f-sans-serif-text;
```

Comment:
You can probably set `color: #000000` here to save repeating it elsewhere (as mentioned above).

More generally, do we need this line? Setting `line-height: 0` would suffice (if that is what you want to achieve).